### PR TITLE
Add accept-permissions flag for install.

### PR DESCRIPTION
**Pull request by anusha-ragunathan**
Add flag for "plugin install" to grant all permissions required for the plugin. This is necessary for unattended installs. Also add basic cli integration tests.

### CODE REVIEW

**Comment by anusha-ragunathan** [comment]
cc @tiborvass

File: api/client/plugin/install.go [hunk 1]
```diff
@@ -13,21 +13,31 @@ import (
...
 		Args:  cli.RequiresMinArgs(1), // TODO: allow for set args
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runInstall(dockerCli, args[0], args[1:])
+			options.name = args[0]
+			return runInstall(dockerCli, options)
 		},
 	}
 
+	flags := cmd.Flags()
+	flags.BoolVar(&options.grantPerms, "grant-permissions", true, "grant all permissions necessary to run the plugin")
```
Comment (tiborvass) [inline api/client/plugin/install.go]: should be `grant-all-permissions`

Comment (anusha-ragunathan) [inline api/client/plugin/install.go]: Done. Also added basic cli integration testing.

**Comment by tiborvass** [comment]
LGTM

**Comment by vdemeester** [comment]
LGTM 🐯

**Comment by thaJeztah** [comment]
we need to think about how to deal with the completion scripts for this; at least to not forget adding them once it leaves experimental

**Pull request merged**